assets/out

```diff
@@ -137,6 +137,9 @@ helm_install() {
     helm_echo="$helm_echo --devel"
   fi
   if [ -n "$version" ]; then
+    if [ -f "$source/$version" ]; then
```

Comment:
you need to same code twice in the two cases (install vs upgrade). Please move it up to around line 56 and resolve $version there.